ethminer/PhoneHome.h

```diff
@@ -18,7 +18,7 @@ class PhoneHome : public jsonrpc::Client
             p.append(param1);
             p.append(param2);
             Json::Value result = this->CallMethod("report_benchmark",p);
-            if (result.isInt())
+            if (result.isIntegral())
```

Comment:
Ah - that was completely accidental commit.

Caused by the PhoneHome.h bug ->https://github.com/ethereum/cpp-ethereum/issues/3234

I will revert that back out, and see if I can work out where that file change is coming from within the build process.